### PR TITLE
Fixing errors and warnings on unit tests

### PR DESCRIPTION
**Pull request by KarinaMendez2**
for: https://app.zenhub.com/workspaces/openstax-unified-5b71aabe3815ff014b102258/issues/openstax/unified/1394

### CODE REVIEW

**Review by PiotrKozlowski — CHANGES_REQUESTED** [review]
About the issue with `PageWithMockedHighlighter.spec.tsx`:
If you add `Highlighter.mock.instances[1].getOrderedHighlights.mockReturnValue(mockHighlights);` in the line `158` then it doesn't throw an error, however, there is still an issue with
```
Warning: An update to %s inside a test was not wrapped in act(...).
...
```

I didn't found why this is happening

File: src/app/components/ScrollLock.tsx [hunk 1]
```diff
@@ -39,14 +39,19 @@ const fadeIn = keyframes`
...
   className?: string;
+  onClick?: () => void;
   mobileOnly?: boolean;
   zIndex?: number;
 }
 
 // tslint:disable-next-line:variable-name
 export const Overlay = styled(({ mobileOnly, zIndex, ...props}: OverlayProps) => {
   useDisableContentTabbing();
-  return <div {...props} />;
```
Comment (TomWoodward) [inline src/app/components/ScrollLock.tsx:49]: you should leave this spread, because the props include everything that you can put on a div

**Review by TomWoodward — CHANGES_REQUESTED** [review]

File: src/app/components/ScrollLock.tsx (hunk 1 above)
Comment (KarinaMendez2) [reply src/app/components/ScrollLock.tsx:49]: the `onClick` was giving a weird console error, refactoring it the way I did fixed it

Comment (KarinaMendez2) [reply src/app/components/ScrollLock.tsx:49]: or `className`, one of those

Comment (KarinaMendez2) [reply src/app/components/ScrollLock.tsx:49]: nvm something from master fixed the issue, changing it back

**Review by TomWoodward — APPROVED** [review]

**Review by PiotrKozlowski — APPROVED** [review]

**Pull request merged**